docs/components/popover.md

```diff
@@ -1,760 +1,515 @@
+    <template #content>
+      Popover content.
+    </template>
+  </KPop>
+</div>
 
 ```html
-<KPop title="Cool header" target=".theme-default-content">
-  <KButton>button</KButton>
+<KPop
+  placement="bottomEnd"
+  button-text="Open popover"
+>
   <template #content>
-    I am a popover, inside the <code>.theme-default-content</code> selector so
-    I can get some of the stylings inside the theme!
+    Popover content.
   </template>
 </KPop>
 ```
 
-### tag
+### trigger
 
-This is the tag that the popover is wrapped around. By default its the div tag.
+Whether popover should be opened on trigger element click or mouseover.
 
-```html
-<KPop title="Cool header" tag="details">
-  <KButton>button</KButton>
-  <template #content>I am inside a &lt;details/&gt; block!</template>
+Accepted values are:
+- `click` (default)
+- `hover`
+
+<KPop
+  trigger="hover"
+  button-text="Open popover"
+>
+  <template #content>
+    Popover content.
+  </template>
 </KPop>
-```
 
-<KPop title="Cool header" tag="details">
-  <KButton>button</KButton>
+```html
+<KPop
+  trigger="hover"
+  button-text="Open popover"
+>
   <template #content>
-    I am inside a &lt;details/&gt; block!
+    Popover content.
   </template>
 </KPop>
+```
 
-### title
+### popoverTimeout
 
-This is the Title of the popover. Either this or the title slot needs to be filled.
+When [`trigger` prop](#trigger) is `hover`, you can provide a timeout for popover to wait before it closes. Default value is 300 milliseconds.
 
-<KPop title="I am a new sample header">
-  <KButton>button</KButton>
+<KPop
+  :popover-timeout="3000"
+  trigger="hover"
+  button-text="Open popover"
+>
   <template #content>
-    I am some sample text!
+    Popover content.
   </template>
 </KPop>
 
 ```html
-<KPop title="I am a new sample header">
-  <KButton>button</KButton>
+<KPop
+  :popover-timeout="3000"
+  trigger="hover"
+  button-text="Open popover"
+>
   <template #content>
-    I am some sample text!
+    Popover content.
   </template>
 </KPop>
 ```
 
-or alternatively, via the slot:
+### disabled
 
-<KPop>
-  <template #default>
-    <KButton>button</KButton>
+Boolean to control whether popover should be disabled. Defaults to `false`.
+
+<KPop
+  disabled
+  button-text="Open popover"
+>
+  <template #content>
+    Popover content.
   </template>
-  <template #title>I am a new sample header</template>
-  <template #content>I am some sample text!</template>
 </KPop>
 
 ```html
-<KPop>
-  <template>
-    <KButton>button</KButton>
+<KPop
+  disabled
+  button-text="Open popover"
+>
+  <template #content>
+    Popover content.
   </template>
-  <template #title>I am a new sample header</template>
-  <template #content>I am some sample text!</template>
 </KPop>
 ```
 
-### trigger
-
-What the popover is triggered by - by default it's triggered on click.
-
-Here are the different options:
+### hideCaret
 
-- `click`
-- `hover`
+Boolean to control whether the popover caret should be visible. Defaults to `true`.
```

Comment:
Are you sure? I feel like the examples in the doc are defaulting to the caret being displayed, not hidden.

![image](https://github.com/Kong/kongponents/assets/67973710/0a88db1b-f558-4eb5-8f37-2e93245accad)

No `:hide-caret="false"` and caret is visible

Reply:
Sorry I'm confused. Are you suggesting that hideCaret prop is not working as expected? Default value is `false` - so **do show caret** unless told otherwise. IN the prop example I am hiding the caret and that works as expected too.
![Screenshot 2024-05-10 at 1 01 08 PM](https://github.com/Kong/kongponents/assets/36751160/61501c90-5f2d-4ec4-919e-50038cc52f01)

Reply:
```suggestion
Boolean to control whether the popover caret should be visible. Defaults to `false`.
```

Yeah, sorry 😅 I meant to call out the text